packages/lib/src/components/internal/Toggle/Toggle.scss

```diff
@@ -0,0 +1,159 @@
+@use 'styles/mixins';
+@import 'styles/variable-generator';
+
+.adyen-checkout-toggle {
+  $component-root: &;
+  $label-padding: token(toggle-label-padding);
+
+  color: inherit;
+  cursor: pointer;
+  display: flex;
+  user-select: none;
+  width: auto;
+
+  @include mixins.box-sizing-setter(true);
+
+  &--disabled {
+    cursor: not-allowed;
+    display: flex;
+  }
+
+  &--readonly {
+    pointer-events: none;
+  }
+
+  &--label-first {
+    align-items: flex-start;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+  }
+
+  &__input {
+    cursor: inherit;
+    opacity: 0;
+    position: absolute;
+  }
+
+  &__track {
+    align-items: center;
+    background-color: token(toggle-track-background-color);
+    border: token(toggle-track-border);
+    border-radius: token(toggle-track-border-radius);
+    display: flex;
+    height: token(toggle-track-height);
+    min-width: token(toggle-track-width);
+    padding: token(toggle-track-padding);
+    position: relative;
+
+    #{$component-root}__input:focus-visible + & {
+      @include mixins.b-focus-ring;
```

Comment:
We probably want to rename this focus ring, to something else. 

There's some Focus state changes coming at some point so it will be easier to refactor.

Reply:
I understand your concern, but I am not aware of the focus state changes that will come up nor have no idea what to rename that to 😅  We are following the bento mixins here.

What is your proposal here?

Reply:
I was just proposal to rename it something `checkout-` prefixed, instead of `b-` from the bento mixins.

Anyway, I see we already use it as the default, my bad.

I will change this, at some point. The changes are the ones we noticed about the button `focus` state not having a good enough outline around it. There's a few components where focus state is like the buttons and it probably needs some love.